exercise/do-your-own-filter.js

```diff
@@ -0,0 +1,134 @@
+/*
+## Do-Your-Own-Filter
+
+### Instructions
+
+1) Create a `Filter` function that takes an array as first argument, a function as second,
+and that works like the method .filter
+
+The use of `filter` is forbidden for this exercise
+
+2) Create a `Reject` function that takes an array as first argument, a function as second,
+and that works like the reject function from lodash.
+
+3) Create a `Partition` function that takes an array as first argument, a function as second,
+and that works like the partition function from lodash.
+
+### Notions
+
+-filter
+https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter
+-reject
+https://lodash.com/docs/4.17.15#reject
+-partition
+https://lodash.com/docs/4.17.15#partition
+
+*/
+
+///*/ // ⚡
+
+///*/// ⚡
+
+let arr1 = [10, -10, 20, -95, 15, 86, 2, 3, 5, 32, 33, 45, 450, 950, 66, 150]
+
+let arrRandom1 = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]
+for (let i = 0; i < 20; i++) {
+  let value = Math.round(Math.random() * 1000 - 1000)
+  arrRandom1.push(value)
+}
+
+const check1 = (el) => el % 2 === 0
+const check2 = (el) => el % 3 === 0
+const check3 = (el) => el % 5 === 0
+const check4 = (el) => el % 10 === 0
+const check5 = (el) => el % 2 === 0 && el % 3 === 0
+const check6 = (el) => el % 5 === 0 && el % 10 === 0
+const check7 = (el) => el % 2 === 0 && el % 5 === 0
+const check8 = (el) => el % 2 === 0 && el % 10 === 0
+const check9 = (el) => el % 3 === 0 && el % 5 === 0
+const check10 = (el) => el % 5 === 0 && el % 10 === 0
```

Comment:
modulo again ? Isn't that a waste of opportunity to show of good use cases of each of those wonderfull functions ?